src/parsing/Parse_rule.ml

```diff
@@ -721,6 +721,39 @@ let parse_validator key env value =
 let parse_validators env key value =
   parse_list env key (parse_validator key) value
 
+(*****************************************************************************)
+(* Supply chain *)
+(*****************************************************************************)
+
+let parse_ecosystem env key value =
+  match value.G.e with
+  | G.L (String (_, (ecosystem, _), _)) -> (
+      match String.lowercase_ascii ecosystem with
```

Comment:
is this meant to be an actual check right now?

Reply:
Oh no, it doesn't need to be